src/ls_server.rs

```diff
@@ -479,11 +484,14 @@ impl Output for StdioOutput {
 
 pub fn run_server(analysis: Arc<AnalysisHost>, vfs: Arc<Vfs>, build_queue: Arc<BuildQueue>) {
     let logger = Arc::new(Logger::new());
+    let logger2 = logger.clone();
+    logger.log(&format!("\nStarting up at time: {}\n", time::now().ctime()));
```

Comment:
Yeah, it's not that important.  Mostly so I can tell if I actually ran it at all, as the timestamp will change.